chia/data_layer/data_layer.py

```diff
@@ -663,34 +663,34 @@ async def make_offer(
                 for our_offer_store in maker
             }
 
-        # Excluding wallet from transaction since failures in the wallet may occur
-        # after the transaction is submitted to the chain.  If we roll back data we
-        # may lose published data.
-
-        wallet_offer, trade_record = await self.wallet_rpc.create_offer_for_ids(
-            offer_dict=offer_dict,
-            solver=solver,
-            driver_dict={},
-            fee=fee,
-            validate_only=False,
-        )
-        if wallet_offer is None:
-            raise Exception("offer is None despite validate_only=False")
-
-        offer = Offer(
-            trade_id=trade_record.trade_id,
-            offer=bytes(wallet_offer),
-            taker=taker,
-            maker=tuple(our_store_proofs.values()),
-        )
+            # Excluding wallet from transaction since failures in the wallet may occur
```

Comment:
I assume comment doesn't apply anything

Reply:
Quite so.  Thanks for catching the details...